types.hh

```diff
@@ -33,6 +33,7 @@
 #include "utils/managed_bytes.hh"
 #include "utils/bit_cast.hh"
 #include "utils/chunked_vector.hh"
+#include "tasks/task_manager.hh"
 
```

Comment:
types.hh is a heavily used header, so I recommend pushing tasks::task_id into tasks/types.hh.